Pentacle Rework

Eqis-Edu opened the pull request:
+ Category Remap - DONE
+ Entry Texts - DONE
+ Pentacles Design - DONE
+ Dummys Design - DONE
+ Chalk Recipes Adjusts - DONE
+ New Materias Recipes - DONE
+ Entities - DONE
+ Resolve Conflicts - DONE

Extras:
+ Add Advancement for getting each chalk
> Used to block entries in the dictionary
+ More progression-based Dictionary
> Except the pentacles category, all is reading conditions, so can skip this with shift+click to unlock all
+ Add Repair Ritual type, and 4 uses
> Chalks, Tools, Armors, Miners
+ Summon Ritual Recipes don't show the ritual dummy as output
> Showing the spawn egg instead (like familiars), also changed color of spirits spawn eggs
+ Update Apotheosis Integration (to match new values used)
> Candles: 4 arcana. Spirit Crystal: 3 eterna, max 45.
+ Creative tab reorder 
> Dictionary, Tools, Books, Chalks, Miners, Crops, Food, Materials, Blocks, Spawn Eggs, Ritual Dummys, Debug items
+ Dictionary does not unlock all categories at once
> Releases when you get the chalk related to the category
+ Dictionary categories reorder
> Started, Spirits, Rituals, Pentacles, Summon, Possess, Familiar, Bind, Storage
+ Adjusts in some possessions
> Skeleton time, Drowned cost and reward, Evoker cost, Elder Guardian cost, Warden cost
+ Chalks item texture update 
> More distinguishable: purple, red. And less saturated: cyan, magenta
+ Golden Sacrificial Bowl redstone signal cloner
> Extract the value under the real bowl using another golden bowl (since most pentacles cover the four sides of the original)   
+ Miners changes
> Now ores has  crying obsidian and gilded blackstone. Basic has deepslate, basalt and blackstone, some weights changed. Enchantable directly on the enchantment table. New Ancient Eldritch Miner for late-game.
+ Changed Familiars advancement tab background to otherplanks texture (instead stone)
+ Fix ritual satchel t2 pentacle
+ Auto gen the Natural Otherworld Sappling item with the block
+ Miners go to output area instead directly breaking in the dimensional mineshaft (server config)
+ Magic Lamp use raw iesnium instead ingot, so as not to get so cheap with high tier crushers
+ Add a gui icon to familiar rituals type to use in dictionary instead the parrot icon
+ Some images in the dictionary have been adjusted
+ Removed extra fire particles from large candles (that I had not been able to remove before)
+ Djinni Weather/Time jobs hitbox size match with model
+ Summon random animal and unbound parrot/drikwing become possessions
+ Ritual to craft Budding Amethyst (mid tier) and Reinforced Deepslate (last tier)
+ Some entities tags now use "c" instead "occultism"


Images:
Dictionary category
![2024-09-25_10 14 13](https://github.com/user-attachments/assets/c31ae31e-6509-4368-bea9-b8a859d3a8df)

All pentacles
![2024-09-25_11 42 32](https://github.com/user-attachments/assets/53799905-117c-4c2c-8d22-64845c2da455)

New items
![2024-09-25_10 16 11](https://github.com/user-attachments/assets/ca10aed5-a79a-4920-974c-dc072262ba0d)

Summon output and dummys
![2024-09-25_10 23 12](https://github.com/user-attachments/assets/aef1c452-6211-4f81-a4f3-dc8d1786e3f7)

Golden Sacrificial Bowl cloning redstone signal (needs an block update, so the lamp and the observer)
![2024-09-25_11 45 27](https://github.com/user-attachments/assets/cd1ac543-d6c3-4079-8746-0caa4b9287ac)

Advancements
![2024-09-25_22 51 10](https://github.com/user-attachments/assets/b8387615-efdb-4ef0-a0b7-ca3b23ad5406)

Eqis-Edu committed:
Repair Ritual type

Eqis-Edu committed:
Change spirits spawn egg color

Eqis-Edu committed:
No more ritual dummy as output (except resurrect)

Eqis-Edu committed:
Update tag SLIMEBALLS -> SLIME_BALLS

Eqis-Edu committed:
Wild drowned don't need silk touch and doubled sniffer egg drop

Eqis-Edu committed:
Ritual Satchel dummy text

Eqis-Edu committed:
Ritual Dummy Types

Eqis-Edu committed:
Fix Summon Ritual give spawn egg

Eqis-Edu committed:
More consistence in ritual type

Eqis-Edu committed:
Update apotheosis integration

Eqis-Edu committed:
Creative tab reorder, real gold -> yellow chalk

Eqis-Edu committed:
Merged natural sapling block and item, book fix

Eqis-Edu committed:
Registry new materials (texture, translation and tags)

Eqis-Edu committed:
Possess Skeleton faster and Wild Evoker cheaper

Eqis-Edu committed:
Add most recipes of chalk materials

Eqis-Edu committed:
Smelting/Blasting Netherite Dust

Eqis-Edu committed:
Repair recipes

Eqis-Edu committed:
Pentacles Remap

Eqis-Edu committed:
Dropout "invoke" return "possess"

Eqis-Edu committed:
Miner to output before break

Eqis-Edu committed:
Ritual Satchel T2 recipe in correct pentacle

Eqis-Edu committed:
Magic lamp cost don't reduce with crusher

Eqis-Edu committed:
Pentacles Remap

Eqis-Edu committed:
(previous) Pentacles CATEGORY Remap.
[now removed unused imports]

Eqis-Edu committed:
config miners to output

Eqis-Edu committed:
Golden Bowl signal cloner

Eqis-Edu committed:
Text to all entries in pentacle category

Eqis-Edu committed:
Broken Sea Heart to Marid Essence

Eqis-Edu committed:
Texture color chalks adjusts

Eqis-Edu committed:
Remove extra fire particle from large candles

Eqis-Edu committed:
Icons and images fix

Eqis-Edu committed:
Chalks advancements

Eqis-Edu committed:
Advancement Conditions in pentacles entries

Eqis-Edu committed:
Entries to category locked until has respective chalk

Eqis-Edu committed:
Changed the "More Chalks" entry

Eqis-Edu committed:
Unbound Marid

Eqis-Edu committed:
Warming Solving

Eqis-Edu committed:
Marid Invulnerabilities

Eqis-Edu committed:
Merge branch 'version/1.21.1' into version/1.21.1

Eqis-Edu committed:
Merge remote-tracking branch 'origin/version/1.21.1' into version/1.21.1

Eqis-Edu committed:
small book changes

Eqis-Edu committed:
Possessed Witch for experience bottle

Eqis-Edu committed:
Goat of Mercy

Eqis-Edu committed:
Possessed Zombified Piglin

Eqis-Edu committed:
Possessed Bee

Eqis-Edu committed:
Eldritch debug

Eqis-Edu committed:
Dont show placeholder dummy

Eqis-Edu committed:
Starting solve compatibility

Eqis-Edu committed:
Real checking

Eqis-Edu committed:
Merge branch 'version/1.21.1' into version/1.21.1

Eqis-Edu committed:
Updated

Eqis-Edu committed:
Small adjustments

Eqis-Edu committed:
Translatable Chalks Advancements

Eqis-Edu committed:
Familiars Advancement Tab Background to Otherplanks

Eqis-Edu committed:
Merge branch 'version/1.21.1' into version/1.21.1

Eqis-Edu committed:
Starting another conflict solving

Eqis-Edu committed:
Merge remote-tracking branch 'origin/version/1.21.1' into version/1.21.1

Eqis-Edu committed:
Starting another conflict solving p2

Eqis-Edu committed:
Merge branch 'klikli-dev:version/1.21.1' into version/1.21.1

Eqis-Edu committed:
Update generated data ritaul recipes

Eqis-Edu committed:
Return "gold" in yellow chalk id

Eqis-Edu committed:
Possess Enderman entry description fix

Eqis-Edu committed:
Fixed some texts that said pentagram instead pentacle

Eqis-Edu committed:
Wild slime hitbox

Heimdallr-1 commented on src/generated/resources/assets/occultism/lang/en_us.json:
```diff
@@ -560,111 +595,225 @@
+  "book.occultism.dictionary_of_spirits.pentacles.red_chalk.name": "The Afrit Control Chalk",
+  "book.occultism.dictionary_of_spirits.pentacles.red_chalk.spotlight.text": "These properties allow for an incredible achievement that has been sought\n for ages before being attained: the summoning of an Afrit bound to the occultist.\n",
+  "book.occultism.dictionary_of_spirits.pentacles.resurrect_spirit.description": "Spirit Resurrection",
+  "book.occultism.dictionary_of_spirits.pentacles.resurrect_spirit.intro.text": "**Purpose:** \"**Purpose:** Resurrect a [#](AA00AA)Familiar[#]()\\\n\\\n**Susjes Simple Circle** is a simple pentacle that returns the physical form of a spirit that has\n been slaughtered, not needing much to do so.\\\n \\\n**Note:** This is the only pentacle where the [#](FF55FF)White Chalk[#]() cannot be replaced  ...
```
"book.occultism.dictionary_of_spirits.pentacles.resurrect_spirit.intro.text": "**Purpose:** \"**Purpose:** Resurrect a [#](AA00AA)Familiar[#]()

Eqis-Edu replied:
Thanks!

Eqis-Edu committed:
Fix double "purpose" text

Eqis-Edu committed:
Fix and conditions some book entries

Eqis-Edu committed:
Merge branch 'klikli-dev:version/1.21.1' into version/1.21.1

Eqis-Edu committed:
Possession entries conditions

Eqis-Edu committed:
Add deepslate, basalt and blackstone to basic miner

Eqis-Edu committed:
Add crying obsidian and gilded blackstone to ore miner

Eqis-Edu committed:
Changes in some basic miner weight

Eqis-Edu committed:
Familiar category progressive unlock (read based)

Eqis-Edu committed:
Creative tab chalks after books

Eqis-Edu committed:
Infusing category progressive unlock (read based)

Eqis-Edu committed:
Summoning category progressive unlock (read based), and turn random animal to possession

Eqis-Edu committed:
Possessed warden cost reduced

Eqis-Edu committed:
Merge branch 'klikli-dev:version/1.21.1' into version/1.21.1

Eqis-Edu committed:
Miners in enchanting table

Eqis-Edu committed:
Ancient Miner (first late-game item)

Eqis-Edu committed:
Craft budding amethyst and reinforced deepslate

Eqis-Edu committed:
Some tags to "c" instead "occultism"

klikli-dev committed:
fix: book links to moved possession rituals

Some possession rituals have been moved into their correct categories, but there were some dangling backlinks.
